charts/bindplane/templates/transform-agent.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if (eq .Values.enterprise true) }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "bindplane.name" . }}-transform-agent
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "bindplane.name" . }}-transform-agent
+    app.kubernetes.io/stack: bindplane
+    app.kubernetes.io/component: transform-agent
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  {{- if eq (include "bindplane.deployment_type" .) "StatefulSet" }}
+  replicas: 1
+  serviceName: {{ include "bindplane.fullname" . }}-transform-agent
```

Comment:
My mistake, I should have suggested that we remove this field as it is specific to StatefulSet